### ISSUE TITLE
[Help] Display start and end date for new events

### ISSUE THREAD

**Issue by van-vothanh** [jquense/react-big-calendar#2254]
Hey everyone, 

I'm trying to find away to display the start and end date/time for newly created events with react-big-calendar. I use the `onSelectSlot` and `soltPropGetter` events to highlight the selected slots but can't figure out how to show the time values.

Any help would be greatly appreciated.

Thanks

**Comment by cutterbl** [comment]
`slotPropGetter` is used to apply a className or styles to a slot. You're styling the selected slots, but that isn't applying any kind of container overlay on those slots, so I'm not sure how you would handle displaying those time values.

**Comment by van-vothanh** [comment]
hey @cutterbl, thanks for getting back to me. I guess my question wasn't clear, I know `slotPropGetter` only deals with styling, what I don't know is whether there's an API somewhere in `react-big-calendar` that would allow such functionality, I would invoke that API in my `onSelectSlot` event handler.

**Comment by cutterbl** [comment]
@van-vothanh I guess I'm misunderstanding your request. It sounded as if you were a) styling selected slots using the `slotPropGetter` and b) want to show the time range of the selection in the actual calendar. Is that not what you were trying to figure out how to accomplish?

**Comment by van-vothanh** [comment]
@cutterbl yes, that's what I want to achieve, I already managed to apply custom styling using `slotPropGetter`, I just don't know how to show the time range of the selected slots. Thanks

**Comment by cutterbl** [comment]
I'm not sure you can. When you select slots they are multiple html elements, not one single item. There is not single control/element that overlays your entire selection area, so no specific element to target for placement of the text.

**Comment by van-vothanh** [comment]
thanks @cutterbl, I do notice an overlay with the times being shown as I make the slot selection, perhaps there's a way to prevent it from going away when i'm done with the selection?

**Comment by cutterbl** [comment]
@van-vothanh Have to verify, but fairly certain that's automatic slot styling of the background color, and not an actual overlay/new element
